staking/types/validator.go

```diff
@@ -109,6 +109,8 @@ type Computed struct {
 	BlocksLeftInEpoch uint64      `json:"-"`
 	Percentage        numeric.Dec `json:"current-epoch-signing-percentage"`
 	IsBelowThreshold  bool        `json:"-"`
+	Epoch             uint64      `json:"epoch"`
+	Block             uint64      `json:"block"`
```

Comment:
In my opinion, it is probably better to change either of the following structures:
https://github.com/harmony-one/harmony/blob/5163c682911deb60566c15bccad5561a6644030a/staking/types/validator.go#L144-L160